tests/5.0/target_teams_distribute/test_target_teams_distribute_reduction_or.c

```diff
@@ -0,0 +1,76 @@
+//===--- test_target_teams_distribute_reduction_or.c-------------------------===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// This test uses the reduction clause on a target teams distribute directive,
+// testing that the variable in the reduction clause is properly reduced using
+// the or operator.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int test_or() {
+  char a[N];
+  double false_margin = pow(exp(1), log(.5)/N);   // See the 'and' operator test for
+  int errors = 0;                                 // an explanation of this math.
+  int num_teams[N];
+  int warned = 0;
+  srand(1);
+
+  for (int x = 0; x < N; ++x) {
+    if (rand() / (double)(RAND_MAX) > false_margin) {
```

Comment:
It might be a matter of changing the order of ths `>`

Reply:
`      if (rand() / (double)(RAND_MAX) < false_margin) {`
This is the code in the and test, it is already flipped. This name is the same, which is misleading so I will change it.